docs/modules/ROOT/pages/explanation/decisions/unit-prefixes.adoc

```diff
@@ -0,0 +1,22 @@
+= Unit prefixes
+
+== Problem
+
+On a low-level we work with bytes.
+Representing bytes to a human isn't practical, therefor a unit prefix has to be used to make it better understandable.
+
```

Comment:
Maybe extend the problem desc that this decision would also apply to product specifications and marketing and maybe other places?

Reply:
Done.